src/browser/BrowserAccessControlDialog.cpp

```diff
@@ -59,3 +59,10 @@ void BrowserAccessControlDialog::setRemember(bool r)
 {
     m_ui->rememberDecisionCheckBox->setChecked(r);
 }
+
+void BrowserAccessControlDialog::setHTTPAuth(bool httpAuth)
+{
+    if (httpAuth) {
+        m_ui->rememberDecisionCheckBox->setEnabled(false);
```

Comment:
I'd prefer hiding it over setting it disabled. Should also simply do:
`m_ui->rememberDecisionCheckBox->setVisible(!httpAuth)`

Reply:
Done.